lib/index.js

```diff
@@ -1178,8 +1178,15 @@ exposeMethod('consume');
  * not provide a new stream for you to push values onto, and it will unsubscribe
  * as soon as it has a single error, value or nil from the source.
  *
- * You probably won't need to use this directly, but it is used internally by
- * some functions in the Highland library.
+ * You can use this method to integrate with other libraries that expect a
+ * function that takes a node style callback as its parameter.
+ *
+ * One of these usages is with promise libraries like Q.
+ *
+ * q.ninvoke(xs.collect(), "pull")
```

Comment:
This doesn't format correctly. The docs is basically markdown. You should just move your code example below the metadata section. Everything down there is formatted as code. Also, can you add a comment along the lines of

    // Use with Q

above your example?